spec/index.bs

```diff
@@ -2428,114 +2428,115 @@ A {{GPUBindGroupLayout}} object has the following internal slots:
+            1. If any of the following conditions are unsatisfied:
+                <div class=validusage>
+                    - If |bindingDescriptor|.{{GPUBindGroupLayoutEntry/visibility}} includes
+                        {{GPUShaderStage/VERTEX}}:
+                        - {{GPUBindingType/storage-buffer}} and {{GPUBindingType/writeonly-storage-texture}}
+                            are not allowed.
+
+                    - If |bindingDescriptor|.{{GPUBindGroupLayoutEntry/type}} is **not**
+                        {{GPUBindingType/"uniform-buffer"}}, {{GPUBindingType/"storage-buffer"}}, or
+                        {{GPUBindingType/"readonly-storage-buffer"}}:
+                        - |bindingDescriptor|.{{GPUBindGroupLayoutEntry/hasDynamicOffset}} is `undefined`.
+                        - |bindingDescriptor|.{{GPUBindGroupLayoutEntry/minBufferBindingSize}} is `undefined`.
+
+                    - If |bindingDescriptor|.{{GPUBindGroupLayoutEntry/type}} is **not**
+                        {{GPUBindingType/"sampled-texture"}}, {{GPUBindingType/"readonly-storage-texture"}},
+                        or {{GPUBindingType/"writeonly-storage-texture"}}:
+                            - |bindingDescriptor|.{{GPUBindGroupLayoutEntry/viewDimension}} is `undefined`.
+
+                    - If |bindingDescriptor|.{{GPUBindGroupLayoutEntry/type}} is **not**
+                        {{GPUBindingType/"sampled-texture"}}:
+                        - |bindingDescriptor|.{{GPUBindGroupLayoutEntry/textureComponentType}} is `undefined.`
+                        - |bindingDescriptor|.{{GPUBindGroupLayoutEntry/multisampled}} is `undefined.`
+
+                    - If |bindingDescriptor|.{{GPUBindGroupLayoutEntry/type}} is **not**
+                        {{GPUBindingType/"readonly-storage-texture"}} or
+                        {{GPUBindingType/"writeonly-storage-texture"}}:
+                        - |bindingDescriptor|.{{GPUBindGroupLayoutEntry/storageTextureFormat}} is `undefined`.
+
+                    - If |bindingDescriptor|.{{GPUBindGroupLayoutEntry/type}} is {{GPUBindingType/sampled-texture}}
+                        and |bindingDescriptor|.{{GPUBindGroupLayoutEntry/multisampled}} is `true`:
+                        - |bindingDescriptor|.{{GPUBindGroupLayoutEntry/viewDimension}} is
+                            {{GPUTextureViewDimension/2d}}.
+
+                    - If |bindingDescriptor|.{{GPUBindGroupLayoutEntry/type}} is
+                        {{GPUBindingType/readonly-storage-texture}} or
+                        {{GPUBindingType/writeonly-storage-texture}}:
+                        - |bindingDescriptor|.{{GPUBindGroupLayoutEntry/viewDimension}} is not
+                            {{GPUTextureViewDimension/cube}}
+                        - |bindingDescriptor|.{{GPUBindGroupLayoutEntry/viewDimension}} is not
+                            {{GPUTextureViewDimension/cube}}{{GPUTextureViewDimension/cube-array}}.
```

Comment:
Currently this says "cubecube-array".

Actually, I think this could just be condensed.
```suggestion
                        - |bindingDescriptor|.{{GPUBindGroupLayoutEntry/viewDimension}} is not
                            {{GPUTextureViewDimension/cube}} or {{GPUTextureViewDimension/cube-array}}.
```